mne/tests/test_spectral_connectivity.py

```diff
@@ -0,0 +1,52 @@
+import os.path as op
+
+import numpy as np
+
+import mne
+from mne import io, pick_types
+from mne.connectivity import spectral_connectivity
+from mne.datasets import testing
+
+
+data_path = testing.data_path(download=False)
+fname_raw_testing = op.join(data_path, 'MEG', 'sample',
+                            'sample_audvis_trunc_raw.fif')
+
+base_dir = op.join(op.dirname(__file__), '..', 'io', 'tests', 'data')
+raw_fname = op.join(base_dir, 'test_raw.fif')
+event_fname = op.join(base_dir, 'test-eve.fif')
+
+raw = io.read_raw_fif(raw_fname)
+events = mne.read_events(event_fname)
+picks = pick_types(raw.info, meg=True, eeg=True, stim=True, ecg=True,
+                   eog=True, include=['STI 014'], exclude='bads')
+
+reject = dict(grad=1000e-12, mag=4e-12, eeg=80e-6, eog=150e-6)
+flat = dict(grad=1e-15, mag=1e-15)
+
+
+@testing.requires_testing_data
+def test_spectral_connectivity():
```

Comment:
this test shouldn't be in its own file in `mne/tests`.  It should be added to `mne/connectivity/tests/test_spectral.py`. See https://mne.tools/dev/install/contributing.html#all-new-functionality-must-have-test-coverage

Reply:
Thank you! We moved the tests into one file